src/view/auction.njk

```diff
@@ -21,7 +21,7 @@
 
         <h1>Trustlines Validator Auction</h1>
 
-        <p class="content-large">The Trustlines validator auction is used to determine the first set of validators for the Trustlines Blockchain. The first set of validators will be valid for nine months. To participate in the auction, a whitelisted Ethereum address has to bid by sending ETH to the auction contract after it has started and before it has ended. The amount must be greater or equal to the current price.</p>
+        <p class="content-large">The Trustlines validator auction is used to determine the first set of validators for the Trustlines Blockchain. The first set of validators will be valid for nine months. To participate in the auction, a whitelisted Ethereum address has to bid by sending TLN to the auction contract after it has started and before it has ended. The amount must be greater or equal to the current price.</p>
```

Comment:
This sounds like you should send the tokens directly, which will lock them. 
We should write it like
"has to bid by approving the TLN token contract and then calling the `bid()` function on the auction contract". This can be done over this website.